contrib/coreos/user-data.example

```diff
@@ -33,6 +33,41 @@ coreos:
       Service=docker.service
       [Install]
       WantedBy=sockets.target
+  - name: flanneld.service
+    command: start
+    content: |
+      [Unit]
+      Description=Network fabric for containers
+      Documentation=https://github.com/coreos/flannel
+      Requires=early-docker.service etcd.service
+      After=etcd.service early-docker.service
+      Before=early-docker.target
+
+      [Service]
+      Type=notify
+      Restart=always
+      RestartSec=5
+      EnvironmentFile=/etc/environment
+      Environment="TMPDIR=/var/tmp/"
+      Environment="DOCKER_HOST=unix:///var/run/early-docker.sock"
+      Environment="FLANNEL_VER=0.4.1"
+      LimitNOFILE=40000
+      LimitNPROC=1048576
+      ExecStartPre=/sbin/modprobe ip_tables
+      ExecStartPre=/usr/bin/mkdir -p /run/flannel
+      ExecStartPre=/usr/bin/touch /run/flannel/options.env
+      ExecStartPre=-/usr/bin/etcdctl mk /coreos.com/network/config '{"Network":"172.17.0.0/16", "SubnetLen": 20, "SubnetMin":"172.17.0.0","SubnetMax":"172.17.208.0", "Backend": {"Type": "vxlan"}}' || true
```

Comment:
I believe we need to change the network range to avoid an overlap with our Vagrant address scheme: `172.17.8.0/24`